versioned_docs/version-2.x/advanced/server/add-http-endpoint.mdx

```diff
@@ -181,3 +181,42 @@ export default {
   },
 };
 ```
+
+## Extend the HTTP server with `onServerInit`
+
+<SinceVersion tag="2.30" />
+
+The `onServerInit` hook is a server module extension point introduced in
+Front-Commerce 2.30. It allows you to extend the Express HTTP server or perform
+additional tasks during server initialization.
+
+To use the `onServerInit` hook, follow these steps:
+
+1. Create a new server module or open an existing one.
+2. In the `module.config.js` file of your server module, define the
+   `onServerInit` function. This function will be called when the server is
+   initialized and will receive an instance of the current server object as a
+   parameter.
+
+```js title="my-module/server/module.config.js"
+export default {
+  onServerInit: (server) => {
+    // Perform server initialization tasks here
+    console.log("Server is initialized");
+  },
+};
+```
+
+3. Save the changes to your `module.config.js` file.
```

Comment:
These steps seem a bit redundant. 

For example `Save the changes to your `module.config.js` file.` brings no value.
and `Create a new server module or open an existing one.` is specified in step `2`

```suggestion
Create a new [server module](/docs/2.x/advanced/server/add-http-endpoint#what-is-a-module) 
or open an existing one. In the `module.config.js` file of your server module, define the
`onServerInit` function. This function will be called when the server is
initialized and will receive an instance of the current server object as a
parameter.

```js title="my-module/server/module.config.js"
export default {
  onServerInit: (server) => {
    // Perform server initialization tasks here
    console.log("Server is initialized");
  },
};
```

You should now see a `"Server is initialized"` console log once your server has started.
```

Reply:
I adapted the content with a slightly different one than suggested, but removed duplication and the useless bullet points.